### ISSUE TITLE
Show "next steps" after kompose convert

### ISSUE THREAD

**Issue by containscafeine** [kubernetes-incubator/kompose#345]
It would be great if we output a message relaying the "what next" information after `kompose convert` is run.

For e.g.
- If a `pvc` object is created, we should display something like "you need to have a `pv` ready to use this object"
- In case of a missing port, "you might want to expose this service manually since kompose could not detect a port"

Thoughts?

**Comment by cdrage** [comment]
I agree with this, @surajnarwade has updated this in https://github.com/cdrage/kompose/commit/f88d48961d1a9db3a6173249f504c8412e2264ea however, I still think there could be some improvement. 

Tools such as Helm have *a lot* of logging output that describes all the steps after launching / using Helm.

**Comment by cdrage** [comment]
Yeah.... we REALLY need to improve this.


My suggestion is that we add some details on what needs to be done (as explained above) such as:
- [ ] Explain `pv` is required
- [ ] Explain what HAS been done
- [ ] Change output to a better way `[NOT CONVERTED] - depends_on` for example. for parts of compose we aren't able to convert

**Comment by cdrage** [comment]
Going to close this for now as honestly, our documentation covers it and this issue is 2 years old.
